_posts/en/newsletters/2023-02-22-newsletter.md

```diff
@@ -0,0 +1,291 @@
+  future payments for a very long time---a much longer duration than the
+  spender may use for nodes that did not advertise high availability.
+  Spenders maximizing payment speed (rather than low fees) would
+  preferentially choose payment paths consisting of self-identified
+  highly available nodes.
+
+    Christian Decker [replied][decker qos] with an excellent summary of
+    problems in reputation systems, including cases of self-proclaimed
+    reputation.  One of his concerns was that typical spenders won't
+    send anywhere near enough payments to frequently encounter the same
+    nodes in a large network of payment channels.  If repeat business is
+    rare anyway, then the threat of temporarily not providing repeat
+    business may not be effective.
+
+    Antoine Riard [reminded][riard boomerang] participants about an
+    alternative approach to speeding up payments---overpayment with
+    recovery.  Previously described as boomerang payments (see
+    [Newsletter #86][news86 boomerang]) and refundable overpayments (see
+    [Newsletter #192][news192 pp]), a spender would take their
+    payment amount plus some extra money, split it into several
+    [parts][topic multipath payments], and send the parts by a variety
+    of routes.  When a sufficient number of parts has arrived to pay the
+    invoice, the receiver claims just those parts and rejects any
+    additional parts (with additional funds) that arrive later.  This
+    requires spenders wanting fast payments to have some additional funds
+    in their channel but it works even if some of the paths selected by the
+    spender fail.  This reduces the need for spenders to be able to
+    easily find highly available channels.  The challenge of this
+    approach is building a mechanism that prevents receivers from
+    keeping any overpayment that arrives.
+
+- **Feedback requested on LN good neighbor scoring:** Carla Kirk-Cohen
+  and Clara Shikhelman [posted][ckc-cs reputation] to the Lightning-Dev
+  mailing list to request feedback on the recommend parameters for how a
+  node should judge whether its channel counterparties are a good source of
+  forwarded payments.  They suggest several criteria to judge by and
+  recommended default parameters for each criterion, but are seeking
+  feedback about the choices made.
+
+    If a node determines that one of its peers is a good neighbor and
+    that neighbor tags a forwarded payment as being endorsed by it, the
+    node can give that payment access to more of its resources than it
+    gives unqualified payments.  The node may also endorse the payment when
+    forwarding it to the next channel.  As described in a prior paper
+    co-authored by Shikhelman (see
+    [Newsletter #226][news226 jam]), this is part of a proposal to
+    mitigate [channel jamming attacks][topic channel jamming attacks].
+
+- **Proposed BIP for Codex32 seed encryption scheme:** Russell O'Connor
```

Comment:
```suggestion
- **Proposed BIP for Codex32 seed encoding scheme:** Russell O'Connor
```

If I understand correctly, it does not encrypt anything.

Reply:
Took this suggestion